seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -235,7 +235,7 @@
         "<strong>Examples:</strong>",
         "<blockquote>var someVariable;<br>var anotherVariableName;<br>var thisVariableNameIsTooLong;</blockquote>",
         "<h4>Instructions</h4>",
-        "Fix the variable declarations and assignments so their names use <dfn>camelCase</dfn>."
+        "Modify the existing declarations and assignments so their names use <dfn>camelCase</dfn>.<br>In other words, do not create any new variables; just fix existing incorrect variable names in the declaration and assignment statements."
```

Comment:
~~In other words, do not create any new variables; just fix existing incorrect variable names in the declaration and assignment statements~~ Do not create any new variables.

Saves from repeating ourselves.